src/openrct2-win/openrct2-win.cpp

```diff
@@ -10,7 +10,7 @@
 // Windows.h needs to be included first
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
-
+#include <memory>
```

Comment:
You don't need these, as you're not using anything from `memory` anymore.

This would also satisfy the code formatting tool. Make sure to revert to having the line break as before.